### ISSUE TITLE
Fail to login NAS via https

### ISSUE THREAD

**Issue by xiajing1980** [LouWii/syno-music-app#2]
The app was stuck in the login screen when I connect my NAS via https protocal.

**Comment by LouWii** [comment]
You're connecting using the IP address ? Local network or over internet ?
I must say, I didn't try connecting using `https` as I only use the app at home, when the NAS is on the local network.

**Comment by xiajing1980** [comment]
I connected my NAS using a DDNS domain name over internet. but the problem still exist when I connect my NAS using IP address over home LAN. if only I use https protocol.

**Comment by LouWii** [comment]
Do you have a valid SSL certificate on your NAS ?

**Comment by xiajing1980** [comment]
Yes, I do use a self-signed certificate. I think that may be the problem.
